src/main/java/org/ngafid/accounts/User.java

```diff
@@ -240,7 +247,7 @@ public static User get(Connection connection, int userId, int fleetId) throws SQ
         user.fleet = Fleet.get(connection, fleetId);
 
         //Get the email preferences for this user
-        user.userEmailPreferences = getUserEmailPreferences(connection, userId);
+        //user.userEmailPreferences = getUserEmailPreferences(connection, userId);
         UserEmailPreferences.addUser(user);
```

Comment:
This method requires a connection, you should just pass it as a parameter. Not sure if it is "best practice" but its usually what we do in the rest of the codebase. Look around and you'll see a lot of methods with `Connection connection` as the first parameter.